lib/commands/project/create.ts

```diff
@@ -45,8 +46,8 @@ export class CreateCommand extends ProjectCommandBaseLib.ProjectCommandBase {
 				this.$fs.deleteDirectory(projectPath).wait();
 				throw err;
 			}
-			
-			if (this.$options.simulator) {
+
+			if (this.$options.simulator && this.$simulatorPlatformServices && this.$simulatorPlatformServices.canRunApplication().wait()) {
```

Comment:
`this.$simulatorPlatformServices` is empty object, so  `if(this.$simulatorPlatformServices)` is true. You should use:
```TypeScript
if (this.$options.simulator && this.$simulatorPlatformServices.canRunApplication && this.$simulatorPlatformServices.canRunApplication().wait()) {
```